amber/convert_amber.py

```diff
@@ -366,7 +369,8 @@ def convert_yaml(yaml_name, ffxml_dir, ignore=ignore):
         source = provenance['Source'] = []
         for source_file in source_files:
             if MODE == 'LEAPRC':
-                _filename = os.path.join(AMBERHOME, 'dat/leap/cmd', source_file)
```

Comment:
We need to generalize this. Perhaps we can first try local path and then try $AMBERHOME if that fails?
```python
if os.path.exists(source_file):
   # use local
else:
   #use AMBERHOME